.github/workflows/integration-tests.yml

```diff
@@ -311,7 +311,7 @@ jobs:
         runner: ${{fromJson(needs.Runner-Preparation.outputs.matrix-HIP)}}
     name: Integration-Tests (${{matrix.runner[1] == 'gfx90a' && 'mi210' || 'mi300x'}})
     container:
-      image: rocm/pytorch:rocm6.1_ubuntu22.04_py3.10_pytorch_2.4
+      image: rocm/pytorch:rocm6.2.2_ubuntu22.04_py3.10_pytorch_release_2.3.0
```

Comment:
I couldn't find rocm 6.2 images with pytorch 2.4. Maybe someone could point me to them.

Reply:
Why do you need pytorch 2.4? 
Previously we need pytorch 2.3 (or 2.2) for fp8 support.

Reply:
I recall some interpreter tests failed due to old pytorch version (which didn't support some types). I cannot recall the exact details though.